index.rst

```diff
@@ -55,7 +55,7 @@ Python Examples
     :caption: NCL
```

Comment:
I realize this was already there, but am wondering if this needs some additional context somewhere.  It leads me to think that this section will be an NCL reference like in NCL.

Reply:
Ha yeah. I am also not totally satisfied with our naming surrounding this section (as also referenced elsewhere in this PR)

Do you have any opinions about a better heading? Maybe something like "NCL to Python"?

Reply:
Yeah, I think NCL to Python is great!